# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/ipc/PWebRenderBridge.ipdl
# Commit: 46c9b479034f
# Full Hash: 46c9b479034f2c6ee710d8d5a703de4a07e77557
# Author: Alexis Beingessner <a.beingessner@gmail.com>
# Date: 2019-07-12 21:43:58
# Regressor Bug: 1547351
# File Overlap Count: 2
# Description:
#   Bug 1547351 - Automatically set content process render root r=kats
#   
#   This implements the idea of automatically setting a content proc's
#   render root based on the render root enclosing the iframe that
#   points to it. There was a bit of cleanup in here that was a bit
# ==============================================================================

diff -r 4689b2870416 -r 46c9b479034f gfx/layers/ipc/PWebRenderBridge.ipdl
--- a/gfx/layers/ipc/PWebRenderBridge.ipdl	Fri Jul 12 14:14:22 2019 +0000
+++ b/gfx/layers/ipc/PWebRenderBridge.ipdl	Fri Jul 12 14:14:22 2019 +0000
@@ -49,14 +49,13 @@
   async DeleteCompositorAnimations(uint64_t[] aIds);
   async SetDisplayList(RenderRootDisplayListData[] displayLists,
                        OpDestroy[] toDestroy, uint64_t fwdTransactionId, TransactionId transactionId,
-                       IdNamespace aIdNamespace, bool containsSVGGroup,
+                       bool containsSVGGroup,
                        VsyncId vsyncId, TimeStamp vsyncStartTime,
                        TimeStamp refreshStartTime, TimeStamp txnStartTime, nsCString txnURL, TimeStamp fwdTime,
                        CompositionPayload[] payloads);
-  async EmptyTransaction(FocusTarget focusTarget, uint32_t aPaintSequenceNumber,
+  async EmptyTransaction(FocusTarget focusTarget,
                          RenderRootUpdates[] renderRootUpdates,
                          OpDestroy[] toDestroy, uint64_t fwdTransactionId, TransactionId transactionId,
-                         IdNamespace aIdNamespace,
                          VsyncId vsyncId, TimeStamp vsyncStartTime,
                          TimeStamp refreshStartTime, TimeStamp txnStartTime,
                          nsCString txnURL, TimeStamp fwdTime,